editor/engine-features/render-config.json

```diff
@@ -14,19 +14,22 @@
             "label": "i18n:ENGINE.features.gfx_webgl.label",
             "description": "i18n:ENGINE.features.gfx_webgl.description",
             "category": "graphics",
-            "enginePlugin": true
+            "enginePlugin": true,
+            "envCondition": "$HTML5 || $MINIGAME"
         },
         "gfx-webgl2": {
             "label": "i18n:ENGINE.features.gfx_webgl2.label",
             "description": "i18n:ENGINE.features.gfx_webgl2.description",
             "category": "graphics",
-            "enginePlugin": true
+            "enginePlugin": true,
+            "envCondition": "$HTML5 || $MINIGAME"
         },
         "gfx-webgpu": {
             "label": "i18n:ENGINE.features.gfx_webgpu.label",
             "description": "i18n:ENGINE.features.gfx_webgpu.description",
             "category": "graphics",
-            "enginePlugin": false
+            "enginePlugin": false,
+            "envCondition": "$HTML5"
```

Comment:
COCOS_RUNTIME also supports webgpu now. Need to change to `$HTML5 || $COCOS_RUNTIME`